engineering/methodology.md

```diff
@@ -76,6 +76,19 @@ teams:
 If there are new design documents, they will be presented at the end of the Leads Meeting
 with a brief explanation and the maintainer is chosen.
 
+Presentation order is:
+
+- Infrastructure
+- Language Analysis
+- Data Retrieval
+- Data Processing
+- Applications
+- Machine Learning
+- Product
+- Engineering management (VP of Eng. and CTO)
+
+Minutes will be taken according to this same order, in round robin.
```

Comment:
One suggestion for keeping track of the order: If the person who took minutes one week merges them on the following Wednesday, part of that job could be to post in the leads channel who the next minute-taker is on the list. They're already keeping tabs on the previous minutes anyway, so they're a logical person to ping the next person.

What do you think?

Reply:
Anyway, I think this change is fine whether or not people like the above suggestion, so I'm approving for now and we can always revisit this if we want to add more.